tests/trestle/core/profile_resolver_test.py

```diff
@@ -431,3 +431,10 @@ def test_profile_resolver_no_params(tmp_trestle_dir: pathlib.Path) -> None:
     catalog_str = catalog.oscal_serialize_json()
     # make sure no moustaches remain that would confuse jinja
     assert '{{' not in catalog_str
+
+
+def test_remote_profile_relative_cat(tmp_trestle_dir: pathlib.Path) -> None:
+    """Test profile resolver with remote profile and import of relative catalog path."""
+    profile_path = 'https://raw.githubusercontent.com/usnistgov/oscal-content/main/nist.gov/SP800-53/rev5/json/NIST_SP-800-53_rev5_LOW-baseline_profile.json'  # noqa E501
+    resolved_cat = ProfileResolver.get_resolved_profile_catalog(tmp_trestle_dir, profile_path)
```

Comment:
This file can potentially get deleted, modified etc. Lets fix it to a particular commit/tag instead (the URL should point to that commit/tag instead).

Reply:
There is one tag available but it does not work due to its href loading xml.  I specified a branch that does work.  Either way - it will need to be updated as oscal evolves.